src/software/proto/messages_robocup_ssl_geometry.proto

```diff
@@ -1,71 +1,77 @@
 syntax = "proto2";
 // A 2D float vector.
-message Vector2f {
-  required float x = 1;
-  required float y = 2;
+message Vector2f
+{
+    required float x = 1;
+    required float y = 2;
 }
 
 // Represents a field marking as a line segment represented by a start point p1,
 // and end point p2, and a line thickness. The start and end points are along
 // the center of the line, so the thickness of the line extends by thickness / 2
 // on either side of the line.
-message SSL_FieldLineSegment {
-  // Name of this field marking.
-  required string name = 1;
-  // Start point of the line segment.
-  required Vector2f p1 = 2;
-  // End point of the line segment.
-  required Vector2f p2 = 3;
-  // Thickness of the line segment.
-  required float thickness = 4;
+message SSL_FieldLineSegment
+{
+    // Name of this field marking.
+    required string name = 1;
+    // Start point of the line segment.
+    required Vector2f p1 = 2;
+    // End point of the line segment.
+    required Vector2f p2 = 3;
+    // Thickness of the line segment.
+    required float thickness = 4;
 }
 
 // Represents a field marking as a circular arc segment represented by center point, a
 // start angle, an end angle, and an arc thickness.
-message SSL_FieldCicularArc {
-  // Name of this field marking.
-  required string name = 1;
-  // Center point of the circular arc.
-  required Vector2f center = 2;
-  // Radius of the arc.
-  required float radius = 3;
-  // Start angle in counter-clockwise order.
-  required float a1 = 4;
-  // End angle in counter-clockwise order.
-  required float a2 = 5;
-  // Thickness of the arc.
-  required float thickness = 6;
+message SSL_FieldCicularArc
```

Comment:
"Cicular" seems to be a misspelling. Even though it was previously misspelled as well, should it be fixed?

Reply:
:eyes: !

This is from the SSL, so we should double check this still exists on their master branch, and open a PR to fix it if necessary. (This if from SSL Vision)

Reply:
yes it exists on their master :grimacing: yikes

Reply:
Then we should keep it as `SSL_FieldCicularArc` until they fix it on their master branch so LGTM :100:

Reply:
Created an issue: RoboCup-SSL/ssl-vision#170